{{cookiecutter.repository_name}}/guild.yml

```diff
@@ -63,6 +72,9 @@
     splits:
       sources:
         - data/splits
+    contested_display:
+      sources:
+        - operations/utilities/contested_display.py
```

Comment:
What is going on?

Reply:
this is the streamlit template, I think I need the actual file there to run the streamlit app?

Reply:
Hmm okay so this was needed to hack around it I guess? :/